RELEASES.md

```diff
@@ -0,0 +1,37 @@
+# Releases
+
+This is a quick overview of how to consume our current build and release [artifacts](docs/development/artifacts.md).
+
+## Current state
+
+Currently, we produce build artifacts as part of our CI workflows ([example](.github/workflows/build_linux_packages.yml)) as well as part of our release. As the project is still not ready for production (see [ROADMAP](ROADMAP.md)) this doc assumes you are already familiar with how to use ROCm. If not - you should not start here, please start at [ROCm](https://github.com/ROCm/ROCm).
+
+## Installing from Tarballs
+
+The quickest way to "install" our artifacts is to leverage the `build_tools/fileset_tool.py artifact-flatten` command. You will need to have a [checkout](README.md#Checkout-Sources) to leverage this tool and a Python environment.
+
+Here's a quick example assuming you copied the all the tar files into `${BUILD_ARTIFACTS_DIR}` to "install" TheRock into `${BUILD_ARTIFACTS_DIR}/output_dir`
```

Comment:
What does "installing" do?

It looks like this only extracts files onto disk, and the `rocminfo` test command below is called directly from that location. Should a user set some environment variables to point their applications at the installation? Or add that location to their `PATH`? Presumably `fileset_tool.py artifact-flatten` does not make such changes itself.

Maybe we should have an installation script that the user can `source therock_rocm_install.sh` or something to help with that.